pkg/model/iam/iam_builder.go

```diff
@@ -765,6 +765,7 @@ func addNodeupPermissions(p *Policy, enableHookSupport bool) {
 	addASLifecyclePolicies(p, enableHookSupport)
 	p.unconditionalAction.Insert(
 		"ec2:DescribeInstances", // aws.go
+		"ec2:DescribeInstanceTypes",
```

Comment:
I was originally a little sad that we couldn't get this from the metadata or DescribeInstances.  But I came round to your point of view, because InstanceTypes are generic data that won't really be sensitive, and if there's one we should get rid of it's probably DescribeInstances :-)